grid/lib/utils.py

```diff
@@ -54,6 +55,11 @@ def _attempt_ipfs_connection(ipfs_addr,
     try:
         api = ipfsapi.connect(ipfs_addr, port)
         return api
+    except ConnectionError:
```

Comment:
Perhaps we could use an environment variable to override the default here? For example:

```python
def get_ipfs_api(mode, ipfs_addr='127.0.0.1', port=5001, max_tries=25):
  ipfs_addr = ipfs_addr if 'IPFS_ADDR' not in os.environ else os.environ['IPFS_ADDR']
```

This way we don't have to try to connect twice, we fallback to the ipfs_addr we were provided and we can document it as a part of the log output in `get_ipfs_api` instead of as a fallback inside `_attempt_ipfs_connection`.